packages/api/src/routers/api/dashboards.ts

```diff
@@ -139,6 +140,20 @@ router.put(
         { new: true },
       );
 
+      // Delete related alerts
+      const deletedChartIds = differenceBy(
```

Comment:
nit: would be great to add int test for this change (in 'routers/api/__tests__`)

Reply:
added test